[INFO   ] [Logger      ] Record log in /home/user/.kivy/logs/kivy_17-09-05_120.txt
[INFO   ] [Kivy        ] v1.10.0
[INFO   ] [Python      ] v3.4.3 (default, Aug  9 2016, 15:36:17) 
[GCC 5.3.1 20160406 (Red Hat 5.3.1-6)]
[INFO   ] [Factory     ] 194 symbols loaded
[INFO   ] [Image       ] Providers: img_tex, img_dds, img_pygame, img_pil, img_gif (img_ffpyplayer ignored)
[INFO   ] [Window      ] Provider: pygame(['window_egl_rpi'] ignored)
libpng warning: iCCP: known incorrect sRGB profile
[WARNING] [WinPygame   ] Video: failed (multisamples=2)
[WARNING] [WinPygame   ] trying without antialiasing
[INFO   ] [GL          ] Using the "OpenGL" graphics system
[INFO   ] [GL          ] Backend used <gl>
[INFO   ] [GL          ] OpenGL version <b'2.1 Mesa 11.1.0 (git-525f3c2)'>
[INFO   ] [GL          ] OpenGL vendor <b'VMware, Inc.'>
[INFO   ] [GL          ] OpenGL renderer <b'Gallium 0.4 on llvmpipe (LLVM 3.7, 256 bits)'>
[INFO   ] [GL          ] OpenGL parsed version: 2, 1
[INFO   ] [GL          ] Shading version <b'1.30'>
[INFO   ] [GL          ] Texture max size <8192>
[INFO   ] [GL          ] Texture max units <18>
[INFO   ] [Window      ] virtual keyboard not allowed, single mode, not docked
[INFO   ] [Text        ] Provider: pygame
[INFO   ] [Clipboard   ] Provider: gtk3(['clipboard_dbusklipper'] ignored)
[CRITICAL] [Cutbuffer   ] Unable to find any valuable Cutbuffer provider.
xclip - FileNotFoundError: [Errno 2] No such file or directory: 'xclip'
  File "/home/user/.local/lib/python3.4/site-packages/kivy/core/__init__.py", line 59, in core_select_lib
    fromlist=[modulename], level=0)
  File "/home/user/.local/lib/python3.4/site-packages/kivy/core/clipboard/clipboard_xclip.py", line 17, in <module>
    p = subprocess.Popen(['xclip', '-version'], stdout=subprocess.PIPE)
  File "/usr/lib64/python3.4/subprocess.py", line 859, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.4/subprocess.py", line 1457, in _execute_child
    raise child_exception_type(errno_num, err_msg)

xsel - FileNotFoundError: [Errno 2] No such file or directory: 'xsel'
  File "/home/user/.local/lib/python3.4/site-packages/kivy/core/__init__.py", line 59, in core_select_lib
    fromlist=[modulename], level=0)
  File "/home/user/.local/lib/python3.4/site-packages/kivy/core/clipboard/clipboard_xsel.py", line 16, in <module>
    p = subprocess.Popen(['xsel'], stdout=subprocess.PIPE)
  File "/usr/lib64/python3.4/subprocess.py", line 859, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.4/subprocess.py", line 1457, in _execute_child
    raise child_exception_type(errno_num, err_msg)

[INFO   ] in run
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
[INFO   ] Tabbed - switch
[INFO   ] Tabbed - on_index
[INFO   ] [Time to on_start] 0.896956 <<<<<<<<
[INFO   ] main - swtich to
[INFO   ] junction screen to id thanks to name
[INFO   ] CS - load
[INFO   ] gui/kivy/uix/ui_screens/history.kv
[INFO   ] before update
[INFO   ] before update
[INFO   ] Tabbed - switch
[INFO   ] 2
[INFO   ] CS - leave
[INFO   ] CS - deactivate
[INFO   ] CS - hide
[INFO   ] CS - enter
[INFO   ] [OSC         ] using <multiprocessing> for socket
[INFO   ] [Base        ] Start application main loop
[INFO   ] [GL          ] NPOT texture support is available
[INFO   ] Tabbed - switch
[INFO   ] [CS - activate ]history
[INFO   ] Tabbed - on_index
[INFO   ] Carousel - Invoices
[INFO   ] Carousel - History
[INFO   ] CS - leave
[INFO   ] CS - deactivate
[INFO   ] CS - hide
[INFO   ] before switch to
[INFO   ] Tabbed - switch
[INFO   ] 2
[INFO   ] CS - enter
[INFO   ] [CS - activate ]history
[INFO   ] Tabbed - switch
[INFO   ] 4
[INFO   ] CS - leave
[INFO   ] CS - deactivate
[INFO   ] CS - hide
[INFO   ] CS - enter
[INFO   ] Tabbed - on_index
[INFO   ] Carousel - Requests
[INFO   ] Carousel - Requests
[INFO   ] [CS - activate ]requests
[INFO   ] CS - load
[INFO   ] gui/kivy/uix/ui_screens/requests.kv
[INFO   ] before update
[INFO   ] [Request - update ] 2.971701
[INFO   ] Request - card
[INFO   ] Request - card
[INFO   ] Request - card
[INFO   ] Request - card
[INFO   ] Request - card
[INFO   ] Request - card
[INFO   ] before update
[INFO   ] Tabbed - switch
[INFO   ] 5
[INFO   ] CS - leave
[INFO   ] CS - deactivate
[INFO   ] CS - hide
[INFO   ] CS - enter
[INFO   ] Tabbed - on_index
[INFO   ] Carousel - Address
[INFO   ] Carousel - Address
[INFO   ] [CS - activate ]address
[INFO   ] CS - load
[INFO   ] gui/kivy/uix/ui_screens/address.kv
[INFO   ] [Base        ] Leaving application in progress...
 Traceback (most recent call last):
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 249, in create_handler
     return eval(value, idmap), bound_list
   File "/home/user/Projects/electrum/gui/kivy/uix/ui_screens/address.kv", line 61, in <module>
     height: blue_bottom.item_height
 NameError: name 'blue_bottom' is not defined
 
 During handling of the above exception, another exception occurred:
 
 Traceback (most recent call last):
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 597, in _apply_rule
     rctx['ids'])
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 254, in create_handler
     cause=tb)
 kivy.lang.builder.BuilderException: Parser: File "/home/user/Projects/electrum/gui/kivy/uix/ui_screens/address.kv", line 61:
 ...
      59:            opacity: 1
      60:            size_hint: 1, None
 >>   61:            height: blue_bottom.item_height
      62:            spacing: '5dp'
      63:            Image:
 ...
 NameError: name 'blue_bottom' is not defined
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 249, in create_handler
     return eval(value, idmap), bound_list
   File "/home/user/Projects/electrum/gui/kivy/uix/ui_screens/address.kv", line 61, in <module>
     height: blue_bottom.item_height
 
 
 During handling of the above exception, another exception occurred:
 
 Traceback (most recent call last):
   File "electrum", line 375, in <module>
     d.init_gui(config, plugins)
   File "/home/user/Projects/electrum/lib/daemon.py", line 292, in init_gui
     self.gui.main()
   File "/home/user/Projects/electrum/gui/kivy/__init__.py", line 62, in main
     w.run()
   File "/home/user/.local/lib/python3.4/site-packages/kivy/app.py", line 829, in run
     runTouchApp()
   File "/home/user/.local/lib/python3.4/site-packages/kivy/base.py", line 504, in runTouchApp
     EventLoop.window.mainloop()
   File "/home/user/.local/lib/python3.4/site-packages/kivy/core/window/window_pygame.py", line 403, in mainloop
     self._mainloop()
   File "/home/user/.local/lib/python3.4/site-packages/kivy/core/window/window_pygame.py", line 289, in _mainloop
     EventLoop.idle()
   File "/home/user/.local/lib/python3.4/site-packages/kivy/base.py", line 339, in idle
     Clock.tick()
   File "/home/user/.local/lib/python3.4/site-packages/kivy/clock.py", line 581, in tick
     self._process_events()
   File "kivy/_clock.pyx", line 367, in kivy._clock.CyClockBase._process_events
   File "kivy/_clock.pyx", line 397, in kivy._clock.CyClockBase._process_events
   File "kivy/_clock.pyx", line 395, in kivy._clock.CyClockBase._process_events
   File "kivy/_clock.pyx", line 167, in kivy._clock.ClockEvent.tick
   File "/home/user/Projects/electrum/gui/kivy/uix/screens.py", line 59, in <lambda>
     Clock.schedule_once(lambda dt: self.dispatch('on_activate'), .25)
   File "kivy/_event.pyx", line 718, in kivy._event.EventDispatcher.dispatch
   File "/home/user/Projects/electrum/gui/kivy/uix/screens.py", line 81, in on_activate
     self.load_screen()
   File "/home/user/Projects/electrum/lib/util.py", line 214, in <lambda>
     return lambda *args, **kw_args: do_profile(func, args, kw_args)
   File "/home/user/Projects/electrum/lib/util.py", line 210, in do_profile
     o = func(*args, **kw_args)
   File "/home/user/Projects/electrum/gui/kivy/uix/screens.py", line 70, in load_screen
     self.screen = Builder.load_file('gui/kivy/uix/ui_screens/' + self.kvname + '.kv')
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 301, in load_file
     return self.load_string(data, **kwargs)
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 382, in load_string
     self._apply_rule(widget, parser.root, parser.root)
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 612, in _apply_rule
     e), cause=tb)
 kivy.lang.builder.BuilderException: Parser: File "/home/user/Projects/electrum/gui/kivy/uix/ui_screens/address.kv", line 61:
 ...
      59:            opacity: 1
      60:            size_hint: 1, None
 >>   61:            height: blue_bottom.item_height
      62:            spacing: '5dp'
      63:            Image:
 ...
 BuilderException: Parser: File "/home/user/Projects/electrum/gui/kivy/uix/ui_screens/address.kv", line 61:
 ...
      59:            opacity: 1
      60:            size_hint: 1, None
 >>   61:            height: blue_bottom.item_height
      62:            spacing: '5dp'
      63:            Image:
 ...
 NameError: name 'blue_bottom' is not defined
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 249, in create_handler
     return eval(value, idmap), bound_list
   File "/home/user/Projects/electrum/gui/kivy/uix/ui_screens/address.kv", line 61, in <module>
     height: blue_bottom.item_height
 
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 597, in _apply_rule
     rctx['ids'])
   File "/home/user/.local/lib/python3.4/site-packages/kivy/lang/builder.py", line 254, in create_handler
     cause=tb)
 
